applications/serial_lte_modem/doc/PPP_AT_commands.rst

```diff
@@ -93,3 +92,25 @@ Example
   #XPPP: 0,0
 
   OK
+
+Testing on Linux
+================
+
+You can test SLM's PPP on Linux by using the ``pppd`` command.
```

Comment:
I think that this would benefit on having the overlay's which you have to use in here as well, or linked to here.

Reply:
Also, do I need to run some AT-commands for CMUX or PPP before doing this? If yes, this should give an example of those commands. I got this to work previously, but cannot remember anymore how, and the documentation leaves this open for interpretation.